src/mock-doc/element.ts

```diff
@@ -297,6 +314,25 @@ export class MockSVGElement extends MockElement {
   getTotalLength(): number {
     return 0;
   }
+  getComputedTextLength(): number {
+    return 0;
+  }
+  createSVGPoint() {
+    return {
+      matrixTransform: defaultSVGMatrix,
+      x: 0,
+      y: 0,
+    };
+  }
+  getBBox() {
+    return { x: 0, y: 0, width: 10, height: 10 };
+  }
+  getCTM() {
+    return defaultSVGMatrix;
+  }
+  getScreenCTM() {
+    return defaultSVGMatrix;
+  }
```

Comment:
These methods appear to not exist on `SVGElement`, but rather it's[ child class `SVGGraphicsElement`](https://developer.mozilla.org/en-US/docs/Web/API/SVGGraphicsElement).  Can we please move these to a (new) `MockSVGGraphicsElement` class that extends `MockSVGElement`? The reason being is that these methods shouldn't be available on `MockSVGElement` (and by extension, any of its child classes except `MockSVGGraphicsElement`)